Godeps/Godeps.json

```diff
@@ -2,9 +2,6 @@
 	"ImportPath": "github.com/google/cadvisor",
 	"GoVersion": "go1.9",
 	"GodepVersion": "v79",
-	"Packages": [
```

Comment:
Why remove this?

Reply:
I dint remove manually. `godep save` did.
godep removed couple of packages which are needed actually. for ex:
`github.com/SeanDolphin/bqschema`

Will sort out.

-Thanks